packages/block-scripts/shared/generate-dist-block-metadata.js

```diff
@@ -12,6 +12,7 @@ export const generateDistBlockMetadata = async (source) => {
 
   const {
     name,
+    displayName,
```

Comment:
`displayName` is part of the `blockprotocol` object, rather than being a `package.json` field, and was already being added to metadata via `blockprotocol` being spread out - is this change required?